webpack.config.js

```diff
@@ -32,6 +32,7 @@ module.exports = (env, argv) => ({
       path.resolve(__dirname, 'app'),
       path.resolve(__dirname, 'docs', 'examples'),
       path.resolve(__dirname, 'node_modules'),
+      path.resolve(__dirname, 'lib', 'vendor')
```

Comment:
What's this used for?

Reply:
I just added it to put local libraries so that I could work on HiGlass while on the plane :)